egov-persister/department-entity-persister.yml

```diff
@@ -32,12 +30,22 @@ serviceMaps:
 
             - jsonPath: $.departmentEntity.*.lastModifiedTime
 
+        - query: INSERT INTO department_entity_relationship(parent_id, child_id, is_true) VALUES (?, ?, ?);
+          basePath: departmentEntityRelationship.*

```

Comment:
Instead of keeping this parallel to the departmentEntity object, make this a sub object of departmentEntity.

Reply:
Both are independent entity as term of class insertion and update.

Reply:
No. These are not independent entities. The relationship is part of the parent.